tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py

```diff
@@ -74,10 +93,9 @@ def compute_sync_committee_participant_reward(spec, state, participant_index, ac
 
 
 @with_all_phases_except([PHASE0, PHASE1])
-@with_configs([MINIMAL], reason="to create nonduplicate committee")
```

Comment:
I think we should to keep this decorator to reduce the iteration times. (CI running out of time now)

Reply:
yes agreed